Core/MIPS/RiscV/RiscVCompFPU.cpp

```diff
@@ -109,8 +109,28 @@ void RiscVJit::CompIR_FAssign(IRInst inst) {
 		break;
 
 	case IROp::FSign:
-		CompIR_Generic(inst);
+	{
+		fpr.MapDirtyIn(inst.dest, inst.src1);
+		// Check if it's negative zero, either 0x10/0x08 is zero.
+		FCLASS(32, SCRATCH1, fpr.R(inst.src1));
```

Comment:
huh, interesting instruction! And I guess likely to be faster than moving the fp reg to a GPR and implementing this through bit operations on that.